src/vos/tests/vts_io.c

```diff
@@ -2634,32 +2637,22 @@ run_io_test(daos_ofeat_t feats, int keys, bool nest_iterators, const char *cfg)
 
 	init_num_keys = VTS_IO_KEYS;
 
-	feats = feats & DAOS_OF_MASK;
-	if ((feats & DAOS_OF_DKEY_UINT64) && (feats & DAOS_OF_DKEY_LEXICAL)) {
-		D_PRINT("Skipping ambiguous ofeat mask\n");
-		return 0;
-	}
-	if ((feats & DAOS_OF_AKEY_UINT64) && (feats & DAOS_OF_AKEY_LEXICAL)) {
-		D_PRINT("Skipping ambiguous ofeat mask\n");
-		return 0;
-	}
```

Comment:
Do the same checks with corresponding DAOS_OT_... types no longer apply ??

Reply:
Right. DAOS_OT_* does not support bit operation like DAOS_OF_*. The checks are not needed any more.